package-lock.json

```diff
@@ -1,31 +1,31 @@
 {
   "name": "@nextcloud/eslint-config",
-  "version": "8.3.0-beta.1",
+  "version": "8.3.0-beta.2",
   "lockfileVersion": 2,
```

Comment:
Maybe do a `npm update` or remove the file and create a fresh one with `npm i` to update the lockfile to version 3 (much smaller by dropping NPM < 7 support, v3 just supports NPM 7+ and is the default for our minimum of NPM 9).